assets/stylesheets/_variables.scss

```diff
@@ -7,17 +7,20 @@ $base-font-size: 16px;
 
 // Colors
 $cr-gray: rgba(151,151,151,1);           // #979797
+$cr-gray-alpha: rgba(151,151,151,.25);   // #979797
 $cr-gray-dark: rgba(115,115,115,1);      // #737373
 $cr-gray-darker: rgba(77,77,77,1);       // #4D4D4D
 $cr-gray-light: rgba(205,205,205,1);     // #CDCDCD
 $cr-gray-lighter: rgba(231,231,231,1);   // #E7E7E7
+$cr-gray-lightest: rgba(244,244,244,1);  // #F4F4F4
```

Comment:
This convention deviates from Bootstrap's [predefined colors](http://getbootstrap.com/customize/#colors). I'd prefer to use the `lighten()` method in places where you need a slightly lighter color of `$gray-light` just to prevent any confusion.